R/render.R

```diff
@@ -25,8 +25,10 @@
 #' @param use_freezer Force use of frozen computations for an incremental
 #'  file render.
 #' @param cache Cache execution output (uses knitr cache and jupyter-cache
-#'   respectively for Rmd and Jupyter input files).
+#'  respectively for Rmd and Jupyter input files).
 #' @param cache_refresh Force refresh of execution cache.
+#' @param override An optional named list used to temporarily override YAML
+#'   metadata.
```

Comment:
A bit clearer would be:

> An optional named list used to override metadata specified in the document front matter and/or the `_quarto.yml` configuration file.

Reply:
Although I just realized: It doesn't seem to be possible to override arbitrary config keys from `_quarto.yml` but only those beneath the chosen output format. 🫤

Reply:
Using `--metadata-file` should solve that. I'll do the modification